### DISCUSSION TITLE
What's the fastest way to build the interface of a SaaS?

### DISCUSSION THREAD

**Discussion by mingolladaniele** [enesien/venefish#3] [General]
Hi! Thanks for your incredible work!

I'm building my first SaaS and I'm actually using Next.js + Tailwind CSS + Shadcn to develop the frontend.
I don't want to spend a lot of time building the interface and I'd like to have some recommendations/resource on where I can find free pre-made templates for a SaaS interface (navbar, user profile page, login, registration, etc.) where I just "copy and paste" and it works.

Thanks :D

**Comment by enesien** [comment]
Hi. Thanks!

Although nothing is just copy/paste, https://saas-ui.dev/ is a good place to start.

There's also some cool UI and templates available here:
- https://htmlrev.com/free-next-templates.html
- https://tailspark.co/templates
- https://tailwindui.com/

I found that looking at the shadcn/ui examples is also really helpful.

Godspeed!
